soc/nxp/imx/imx9/imx95/m7/soc.c

```diff
@@ -53,6 +56,13 @@ static int soc_init(void)
 		}
 	}
 
+	cpu_cfg.cpu_id = CPU_IDX_M7P;
+	cpu_cfg.sleep_mode = CPU_SLEEP_MODE_RUN;
+
+	ret = scmi_cpu_sleep_mode_set(&cpu_cfg);
```

Comment:
This only gets enabled when `CONFIG_ETH_NXP_IMX_NETC` is enabled. Please move outside the if statement.
Also you've to guard it with `CONFIG_NXP_SCMI_CPU_DOMAIN_HELPERS` otherwise compilation fails.

Reply:
Thanks for your commit, have fixed.